llama_stack/providers/remote/inference/cerebras/__init__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from .cerebras import CerebrasInferenceAdapter
```

Comment:
this import must be conditional so that when we load configs we don't need dependencies